documentation/modules/ROOT/pages/021-initial-dir-and-files.adoc

```diff
@@ -133,7 +154,9 @@ EOF
 [.console-input]
 [source,console]
 ----
-export ADMIN_DEV_PASSWORD=<CHANGE_ME>
+export HISTCONTROL=ignoreboth
+ export ADMIN_DEV_PASSWORD=<CHANGE_ME>
+ export ADMIN_PRO_PASSWORD=<CHANGE_ME>
```

Comment:
Remove the first character (extra space) in the lines 158 and 159

Reply:
This change is a suggestion to avoid adding the passwords to bash history

Reply:
You are right, we could use `read -p "question: " variable` for these inputs